Empire/Empire.cpp

```diff
@@ -68,6 +69,9 @@ const std::string& Empire::Name() const
 const std::string& Empire::PlayerName() const
 { return m_player_name; }
 
+bool Empire::IsAuthenticated() const
```

Comment:
does `m_authenticated` make sense as part of empire, and not of the server players list?

Reply:
To use it in multiplayer lobby I need to have it in `SaveGameEmpireData` which populated from `Empire`. Also I can't use server player list because after #2379 it could be possible someone would be disconnected and its authentication status would be unreachable.